pnpm-lock.yaml

```diff
@@ -21910,7 +21910,7 @@ packages:
   /@types/concat-stream/1.6.1:
     resolution: {integrity: sha512-eHE4cQPoj6ngxBZMvVf6Hw7Mh4jMW4U9lpGmS5GBPB9RYxlFg+CHaVN7ErNY4W9XfLIEn20b4VDYaIrbq0q4uA==}
     dependencies:
-      '@types/node': 18.19.1
+      '@types/node': 20.10.5
```

Comment:
You're gonna want to undo these changes, it's a weird local repo bug. Looks like it's everywhere. @jason-ha fyi, it looks like it's not just my machine that's doing it. We're going to accidentally slip in `20.10.5` for some reason because not everyone is paying attention to their `pnpm.yaml` files

Reply:
Or fast follow up to just revert `@types/node` to `18.19.1`, not sure, it's a bug in our local repos.

Reply:
I am still interested in exact repro steps if anyone has them. It is very easy not to follow the pattern that we know leads to good results.
1. from main commit: `pnpm i --frozen-lockfile`
2. make manual package.json changes (do not use `pnpm add`)
3. `pnpm i`
There is an optional extra `pnpm store prune` that could be done for a little more cleanliness, but so far with FF I haven't seen anyone need to go there. In past that was useful when there we complicated feeds in use, which I don't think is the case for FF.

Reply:
All I did is "pnpm i" for your steps 1 & 3. I'm not aware of requirement to use --frozen-lockfile, if that's a requirement. Maybe it's mistake on my part, but I'd rather have some simple command here (alias) instead of relying on remembering to run with this option.

Reply:
I think the flow for most devs was `pnpm i` with the expectation that everything will work. We either need to re-educate everyone or make the current flow work. Doesn't matter to me